node/tests/swamp/sync_test.go

```diff
@@ -0,0 +1,53 @@
+package swamp
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/onsi/gomega"
```

Comment:
I don't like this dependency because the methods aren't documented / easy to follow. Can we please stick to assert / require for now?

Reply:
Agreed. Removed for now